common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -968,7 +969,13 @@ def get_answer(self, _data):
                 new_answer = {answer_id: answers[answer_id]}
             new_answers.update(new_answer)
 
-        return {'answers': new_answers}
+        return {
+            'answers': new_answers,
+            'correct_status_html': self.runtime.render_template(
```

Comment:
The other places where this template is used, it is included in a server-side template. However, "show answer" just sends back some information to the client, and the pre-existing HTML is updated instead of the complete HTML being re-rendered. I need to inject this status HTML for the correct answers (in the case of multiple choice or checkbox), and I don't want to duplicate the definition of the HTML on the client side. Hence I am rendering the fragment and sending it to the client.

I will add a comment in the code.